src/Widgets/DiskGrid.vala

```diff
@@ -19,21 +19,35 @@
  */
 
 public class Installer.DiskButton : Gtk.ToggleButton {
-    public Disk disk { get; construct; }
-    public DiskButton (Disk disk) {
+    public unowned Distinst.Disk disk { get; construct; }
+    public DiskButton (Distinst.Disk disk) {
         Object (disk: disk);
     }
 
     construct {
         margin = 12;
         get_style_context ().add_class (Gtk.STYLE_CLASS_FLAT);
 
-        var disk_image = new Gtk.Image.from_icon_name (disk.get_icon_name (), Gtk.IconSize.DIALOG);
+        // Drives are identifiable by whether they are rotational and/or removable.
+        string icon_name = null;
+        if (disk.is_removable ()) {
+            if (disk.is_rotational ()) {
+                icon_name = "drive-harddisk-usb";
+            } else {
+                icon_name = "drive-removable-media";
+            }
+        } else if (disk.is_rotational ()) {
+            icon_name = "drive-harddisk-scsi";
+        } else {
+            icon_name = "drive-harddisk-solidstate";
```

Comment:
It looks like these aren't icons provided in either Pop! or elementary, but I think they'd make sense. We can coordinate with @danrabbit and [@elementary/icons](https://github.com/elementary/icons) for the exact icon names, then propagate that to [@pop-os/icon-theme](https://github.com/pop-os/icon-theme).

Reply:
The solidstate icons are in Adwaita, so issues have been opened on each repo for that one.